### DISCUSSION TITLE
Docker compose?

### DISCUSSION THREAD

**Discussion by stolirocks1** [sct/overseerr#262] [Feature Requests]
Docker compose, can anyone make one? Thanks

**Comment by tzw9373** [comment]
Here's mine, composerize.com is a great tool to generate your own. 

version: "3.5"
services:  
  overseerr:
        container_name: overseerr
        environment:
            - LOG_LEVEL=info
            - TZ=Amercia/Chicago
        ports:
            - 5055:3000
        volumes:
            - /home/my_username/overseerr/config:/app/config
        restart: unless-stopped
        image: sctx/overseerr
